### PR TITLE
GCW-3247 Fix Unexpected Error Message

### PR DESCRIPTION
**Pull request by bharat619**
### Ticket Link: 
https://jira.crossroads.org.hk/browse/GCW-3247

### What does this PR do?
Fixes the error message while creating item.

### CODE REVIEW

File: app/routes/items/new.js [hunk 1]
```diff
@@ -162,6 +162,5 @@ export default AuthorizeRoute.extend(GradeMixin, {
     controller.set("defaultCondition", this.getDefaultCondition());
     controller.set("valueHkDollar", +defaultValue.value_hk_dollar);
     controller.set("defaultValueHkDollar", +defaultValue.value_hk_dollar);
-    this.setupPrinterId(controller);
```
Comment (bharat619) [inline app/routes/items/new.js:165]: removed this as it is also called [here](https://github.com/crossroads/stock.goodcity/pull/854/files#diff-582238b7d4945ca1309bf7b22d97b16bR74)

File: config/environment.js [hunk 2]
```diff
@@ -94,7 +94,6 @@ module.exports = function(environment) {
         "packages_location",
         "stockit_contact",
         "goodcity_request",
-        "donor_condition",
```
Comment (abulsayyad123) [inline config/environment.js:97]: why removed donor_condition ?

**Review by patrixr — APPROVED** [review]

File: config/environment.js (hunk 2 above)
Comment (patrixr) [reply config/environment.js:97]: same question

Comment (bharat619) [reply config/environment.js:97]: because the donor_condition is being unloaded from store. And it dosn't actually needs to be unloaded

**Review by swatijadhav — APPROVED** [review]

**Review by abulsayyad123 — APPROVED** [review]

**Review by shreyas098 — APPROVED** [review]

**Pull request merged**